examples/seq2seq/README.md

```diff
@@ -43,18 +43,24 @@ python examples/seq2seq/run_summarization.py \
     --model_name_or_path t5-small \
     --do_train \
     --do_eval \
-    --dataset_name xsum \
+    --dataset_name cnn_dailymail \
+    --dataset_config "3.0.0" \
+    --source_prefix "summarize: " \
     --output_dir /tmp/tst-summarization \
     --per_device_train_batch_size=4 \
     --per_device_eval_batch_size=4 \
     --overwrite_output_dir \
     --predict_with_generate \
-    --max_train_samples 500 \
-    --max_val_samples 500
+    --max_train_samples 50 \
+    --max_val_samples 50
```

Comment:
Is it precised somewhere that those options are just to run quick tests?

Reply:
You are asking whether it's documented anywhere in the READMEs why we propose to use those options in the examples, correct?

Yes, this is a good point to clarify.

If so  - would you like me to add the clarification to each README.md or just `examples/README.md?

If not please clarify what you meant.

Thank you.

Reply:
I don't think those are used in any other example README, which provide commands to train on the full datasets. So I think it's only in this one that we would need to say what those do.

We can also not put those arguments in this README and a mention on the main examples/README that all the commands given as examples can be transformed in shorts tests by adding `--max_train_samples 50  --max_val_samples 50`, what do you think?

Reply:
In my experience, I almost always find that it's worth doing a short quick run on all stages first to verify that all stages work. Otherwise there are cases where the training runs for an hour and then crashes in saving the model, or some other event at the end of train, or eval has some problem. 

So unless I'm already familiar with the peculiarity of a given script making sure it can run fully is usually my first go-to approach. 

But also I have a peculiar need where I constantly run very quick tests when I do integration work. So it's possible that this is uncommon.

It's hard for me to tell what other's preference/modus operandi is.

I'm not attached to either way for examples. So if you think it's just simpler to remove those and put those in `examples/README.md` I'm totally happy to do that as you suggested.

Reply:
Yes, having the option to launch a short test for all scripts is really great (thanks for supervising this by the way :-) ) but I think we should leave the commands for full training in each individual README and document the short test in `examples/README/md`, if you're fine with it.

I can also do it as a follow-up since it's a bit out of scope of this PR, just let me know how you prefer to play this!

Reply:
Sure, that's a good suggestion. 

Let's do it here, we are tweaking text-only anyway, so it's much easier to pile things up a bit ;)

I will make an update shortly.